model/task.js

```diff
@@ -23,7 +23,7 @@ var pa11y = require('pa11y');
 // Task model
 module.exports = function (app, callback) {
 	app.db.collection('tasks', function (err, collection) {
-		collection.ensureIndex({name: 1, url: 1, standard: 1}, {w: -1});
+		collection.ensureIndex({name: 1, url: 1, standard: 1, timeout: 1}, {w: -1});
```

Comment:
We probably don't need to index the `timeout` property as we're unlikely to query based on it